package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-storefront",
-  "version": "8.0.0",
+  "version": "8.0.1",
```

Comment:
Not sure why this isn't showing up as a conflict, but 8.0.1 is already released so this should be 8.0.2.

Reply:
resolved